### PR TITLE
Sort apiserver certificates according to kube docs.

### PR DESCRIPTION
**Pull request by ffilippopoulos**
https://kubernetes.io/docs/reference/setup-tools/kubeadm/implementation-details/

### CODE REVIEW

File: worker.tf [hunk 1]
```diff
@@ -7,6 +7,7 @@ data "template_file" "worker-cfssl-new-cert" {
   template = "${file("${path.module}/resources/cfssl-new-cert.sh")}"
 
   vars {
+	  cert_name   = "node"
```
Comment (george-angel) [inline worker.tf]: fmt?

**Comment by george-angel** [comment]
Tested in env?

File: worker.tf (hunk 1 above)
Comment (ffilippopoulos) [reply worker.tf]: sure, fixing..

**Comment by ffilippopoulos** [comment]
runs currently (fine) on `exp-1-gcp` but after some testing I think we also need to use a different `master-kubeconfig` for master kubeletes

**Pull request closed without merging**